src/Platform/Microsoft.Testing.Extensions.VSTestBridge/Capabilities/VSTestBridgeExtensionBaseCapabilities.cs

```diff
@@ -9,6 +9,10 @@ namespace Microsoft.Testing.Extensions.VSTestBridge.Capabilities;
 /// <summary>
 /// The VSTest bridged test framework capabilities.
 /// </summary>
+// NOTE: MSTest no longer uses this, as we don't want to use the vstestProvider.
+// Only NUnit and Expecto use this.
+// https://github.com/nunit/nunit3-vs-adapter/blob/3d0f824243aaaeb85621d3c7dddc92e7a7c45097/src/NUnitTestAdapter/TestingPlatformAdapter/TestApplicationBuilderExtensions.cs#L20
+// https://github.com/YoloDev/YoloDev.Expecto.TestSdk/blob/0d1a3eadd65b605f61bb01d302f28382be76b8ac/src/YoloDev.Expecto.TestSdk/TestApplicationHelpers.fs#L16
 public sealed class VSTestBridgeExtensionBaseCapabilities : ITrxReportCapability, IVSTestFlattenedTestNodesReportCapability, INamedFeatureCapability
```

Comment:
Maybe mark it as obsolete? Expecto and NUnit auhtors are super friendly so if we do the PR they would update.

Reply:
The eventual goal for me at least is to deprecate the whole VSTestBridge.

For now we can't mark as obsolete because of NUnit at least. I have an open issue there, so until it's resolved we will keep this.

Reply:
The eventual goal for me at least is to deprecate the whole VSTestBridge.

For now we can't mark as obsolete because of NUnit at least. I have an open issue there, so until it's resolved we will keep this.